copter/source/docs/systemid-mode-operation.rst

```diff
@@ -95,3 +95,101 @@ main loop frequency divided by a sub-sample factor.
 | 0            |    0         | 8             |
 +--------------+--------------+---------------+
 
+Identification of a Multicopter
+===============================
+Description of the Approach
+---------------------------
+
+The Parameter :ref:`SID_AXIS<SID_AXIS>` of the system identification mode feeds in the frequency-sweep at different spots inside Ardupilot's controlers.
+The locations of these injection points are shown in the following :ref:`figure<fig-ctrl-sys-ardupilot>` as small chirp symbols.
+At each injection point, the sweep can be added to either the roll, pitch or yaw axis. 
+
+.. image:: ../images/ControlSystem.png
+:name: fig-ctrl-sys-ardupilot
+
+For the purpose of obtaining a dynamic model of only the copter itself (aka plant model), it is first necessary to calculate a transfer function(s) between the control inputs of the copter and its measurable dynamic response.
+In this case, the angular rate controller outputs (:math:`p_{Thrust}`, :math:`q_{Thrust}` and :math:`r_{Thrust}`) represent the control inputs of the copter whereas the gyroscope's measured angular rates represent the system response.
+Since a common application of the created system model consists in optimizing the control parameters, the filtered gyroscope signals are used instead of the raw gyroscope data, as these are the direct inputs to the angular rate controller.
+Thereby, the separate modelling of the filters becomes redundant, which makes it possible to directly connect a model of the angular rate controller to the plant model without any further effort.
+
+.. image:: ../images/RateControl.png
+:name: fig-rate-ctrl
+
+The generation of flight data for the system identification is conducted during hovering flight.
+Therefore, the three axes (namely roll, pitch and yaw) are considered as decoupled.
```

Comment:
Is the intent of telling the reader to conduct the testing in hovering flight to ensure an uncoupled response? You may want to add a sentence that says the flight data will be generated at the trim conditions of interest.  So that could be at some forward flight speed or in a hover. But you chose hover because it was easier to model due to an uncoupled response.  however I think multicopters are uncoupled even at some flight speed.  Helicopters  are completely different story ;-)